### ISSUE TITLE
VS Code not starting when offline after update

### ISSUE THREAD

**Issue by steinmetzdenver** [Microsoft/vscode#66041]
Issue Type: <b>Bug</b>

After latest update, VS Code will not open when offline.  Running as administrator.  Verbose shows:
this.handler = ^new channel....

VS Code version: Code 1.30.1 (dea8705087adb1b5e5ae1d9123278e178656186a, 2018-12-18T18:12:07.165Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i3-7100U CPU @ 2.40GHz (4 x 2400)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|7.92GB (4.47GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-babel-coloring|dza|0.0.4
LiveServer|rit|5.3.1

</details>

**Comment by abrahamdev22** [comment]
any updates for this visual code issue?

**Comment by steinmetzdenver** [comment]
The only response I got was to refer me to a long list of other users with
the same problem but offered no real help.
I tried the VSC beta "insider" version on their website and that works fine
offline.  Obviously something is wrong with the official update.
Seems like a lot of people are spending a lot of time and effort on this,
thinking the problem is with their system instead.


Le lun. 7 janv. 2019 à 1:17 AM, abrahamdev22 <notifications@github.com> a
écrit :
